Gemfile

```diff
@@ -18,7 +18,7 @@ group :test do
   gem "capybara"
   gem "evergreen"
   gem "capybara-webkit"
-  gem 'selenium-webdriver'
+  gem 'selenium-webdriver', '2.53.4'
```

Comment:
via this (https://docs.travis-ci.com/user/firefox/) you should be able to select the version of firefox that is compatible with recent versions of `selenium-webdriver`